Add 11.24 release notes (#388)
mondoohq/docs: releases/2024-10-01-mondoo-11.24-is-out.md

---
slug: mondoo-11.24-is-out/
title: Mondoo 11.24 is out!
description: Announcing the 11.24 release of Mondoo with an all-new AWS serverless integration, macOS Sequoia support, and more!
authors: [tim]
image: /img/releases/2024-10-01-mondoo-11.24-is-out/integration.png
tags: [release, mondoo]
---

## 🥳 Mondoo 11.24 is out! This release includes an all-new AWS serverless integration, macOS Sequoia support, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

> ### All new AWS Serverless integration experience

Reviewer: ```suggestion
### All-new AWS serverless integration experience
```

> Over the past few months, our engineers have been hard at work on a complete rewrite of our AWS Serverless integration. This new release builds on our previous design incorporating valuable insights we've gained from securing AWS accounts at scale. What can you expect from this new release?

Reviewer: ```suggestion
Over the past few months, our engineers have been hard at work on a complete rewrite of our AWS serverless integration. This new release builds on our previous design and incorporates valuable insights we've gained from securing AWS accounts at scale. What can you expect from this new release?
```

> - **Enhanced Security** Our new design drastically reduces the number of privileges required to run in your environment so you can gather critical security insights without introducing new risks to your business.

Reviewer: ```suggestion
- **Enhanced security** Our new design drastically reduces the number of privileges required to run in your environment so you can gather critical security insights without introducing new risks to your business.
```

> - **Expanded Scalability** From the smallest dev account to enterprise-sized production accounts, Mondoo has you covered.

Reviewer: ```suggestion
- **Expanded scalability** - From the smallest dev account to enterprise-sized production accounts, Mondoo has you covered.
```

> - **Granular Results** Our new approach to scanning provides resource-level scan granularity in more situations, giving you enhanced visibility into critical risks to your business.

Reviewer: ```suggestion
- **Granular results** - Our new approach to scanning provides resource-level granularity in more situations, giving you enhanced visibility into critical risks to your business.
```

![AWS Serverless integration status page](/img/releases/2024-10-01-mondoo-11.24-is-out/integration.png)

### macOS Sequoia support

> Is your team eager to upgrade to the latest macOS Sequoia release? Now you can keep your bleeding edge users happy and secure with Mondoo! With tested packages, EOL detection support, and new CIS benchmarks you'll be ready to tackle early adopters day one. For users still running Sonoma (14) or Ventura (13), updated CIS benchmark policies offer new and improved security recommendations.

Reviewer: ```suggestion
Is your team eager to upgrade to the latest macOS Sequoia release? Now you can keep your bleeding edge users happy and secure with Mondoo! With tested packages, EOL detection support, and new CIS benchmarks, you'll be ready to tackle early adopters on day one. For users still running Sonoma (14) or Ventura (13), updated CIS benchmark policies offer new and improved security recommendations.
```
